edx_exams/apps/api/v1/views.py

```diff
@@ -400,13 +428,42 @@ class ExamAttemptView(ExamsAPIView):
 
     PUT Response Values
         {'exam_attempt_id': <attempt_id>}: The attempt id of the attempt being updated
+
+    HTTP POST
+    Creates a new attempt based on a provided exam_id
+
+    POST data Parameters
+        'exam_id': The unique identifier for the exam
+        'start_clock': Boolean value representing whether or not the attempt should immediately transition to 'started'
+
+    POST Response Values
+        {'exam_attempt_id': <attempt_id>}: The attempt id of the attempt that was created
     """
 
     authentication_classes = (JwtAuthentication,)
 
+    def get(self, request, attempt_id='None'):
```

Comment:
What's the purpose of the attempt_id here? It doesn't look like you're using it, and you have removed it from the URL pattern. I think you can remove it, because you're using the `user_id` to get an attempt.